eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/mob/ButcherArgument.java

```diff
@@ -43,7 +43,7 @@ public ParseResult<Integer> parse(Invocation<CommandSender> invocation, String a
 
     @Override
     public SuggestionResult suggest(Invocation<CommandSender> invocation, Argument<Integer> argument, SuggestionContext context) {
-        return IntStream.range(0, 6)
+        return IntStream.range(1, 6)
```

Comment:
to nie powinno być config state?

Reply:
Dałem propozycje rozwiązania:

https://github.com/EternalCodeTeam/EternalCore/assets/65517973/6331b41f-5f3f-4514-9a7d-f333e71629f4

Reply:
Tylko nie wiem co jest, ale zwraca mi caly czas invalid usage w tych checkach w argumencie zamiast normalnej wiadomosci